new-standalone-service.sh

```diff
@@ -0,0 +1,108 @@
+#!/bin/bash
+
+cleanup() {
+    # TODO(mohit): Maybe add a trap for SIGINT.
+    rm -rf template.zip
+}
+
+info() {
+    MESSAGE=$1
+    echo $MESSAGE
+}
+
+debug() {
+    if [[ -z "${DEBUG// }" ]]; then
```

Comment:
Out of curiosity, what do the extra slashes/space do? I've just been using e.g. `-z "${DEBUG}"`